### ISSUE TITLE
CTRL + Z ?????

### ISSUE THREAD

**Issue by Mr-Fox-h** [PanosK92/SpartanEngine#52]
Hi
I have used your game engine. it's so good but I can't undo anything.

**Comment by PanosK92** [comment]
True, this is low priority as I have to work on the D3D12 backend, polish Vulkan, add ray tracing and so on.  I'll keep this open as I will tackle it at some point.

**Comment by PanosK92** [comment]
A start has been made: https://github.com/PanosK92/SpartanEngine/blob/master/runtime/Commands/CommandStack.h

It currently works for entity positioning, rotating and scaling.
